easybuild/easyconfigs/a/AMD-BLIS/AMD-BLIS-1.2-GCC-7.3.0-2.30.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'AMD-BLIS'
+version = '1.2'
```

Comment:
@migueldiascosta How about using a `versionsuffix` like `-amd` rather than a different software name?

Makes sense to me if it's closely following upstream...

Reply:
Was told this from AMD, via a vendor: "All AMD optimizations are eventually merged with BLIS/libflame public branches. For bleeding edge code, customers are recommended to tap into current AMD branch."

So yes, using a `versionsuffix` seems to make more sense